cmake/modules/AddSwift.cmake

```diff
@@ -536,7 +536,7 @@ endfunction()
 #
 # source1 ...
 #   Sources to add into this library
-function(_add_swift_library_single target name)
+function(_add_swift_library_single target module_target name)
```

Comment:
Can you add documentation to the header for what module_target is. Also, even better, can you change these options here to be cmake_parse_argument Arguments. Then below when you use "" for the module_target name, it would be at least self documenting that it is not a bug (see question below).